deployment/helm/values.yaml

```diff
@@ -101,11 +101,11 @@ edgex:
       pullPolicy: IfNotPresent
     bootstrapper:
       repository: edgexfoundry/security-bootstrapper
-      tag: "2.2.0"
+      tag: "latest"
```

Comment:
There is no "edgexfoundry/security-bootstrapper:latest" on Dockerhub.

Please change "respository" to point to the Nexus images.

See https://github.com/edgexfoundry/edgex-compose/blob/main/compose-builder/.env